modules/kubernetes/logs/scrubs/json-empties.river

```diff
@@ -0,0 +1,33 @@
+/*
+Module: scrub-json-empties
+Description: Removes any json properties with empty values i.e. "foo": "", "bar": [], "baz": {}
+Arguments:
+  forward_to: Module to forward the output to
+*/
+argument "forward_to" {
+  optional = false
+}
+
+export "process" {
+  value = loki.process.scrub_json_empties
+}
+
+loki.process "scrub_json_empties" {
+  forward_to = [argument.forward_to.value]
+
+  // check logs.agent.grafana.com/scrub-empties annotation, if true remove any json property whose value is set to
```

Comment:
This might also be a great comment to add to the description since it explains the why